goon/code/datums/browserOutput.dm

```diff
@@ -202,8 +203,7 @@ var/list/chatResources = list(
 	return "pong"
 
 /datum/chatOutput/proc/debug(error)
-	error = "\[[time2text(world.realtime, "YYYY-MM-DD hh:mm:ss")]\] Client : [owner.key ? owner.key : owner] triggered JS error: [error]"
-	chatDebug << error
+	rustg_log_write(CHAT_DEBUG_LOG, "Client : [owner.key || owner] triggered JS error: [error][GLOB.log_end]")
```

Comment:
Could you also split it by round and put it in `SetupLogs()` in `world/New()`

It cuts down on bigass files, and also means denghis can do parsing magic on it per round.